### ISSUE TITLE
@context in batch upsert is not accepted

### ISSUE THREAD

**Issue by jason-fox** [Fiware/context.Orion-LD#387]
I make a batch upsert request placing the `@context` within the body of the request:

```console
curl -L -X POST 'http://localhost:1026/ngsi-ld/v1/entityOperations/upsert/' \
-H 'Content-Type: application/ld+json' \
-H 'fiware-service: SmartGondor' \
--data-raw '[
        {
            "@context": "http://context.json-ld",
            "pressure": {
                "type": "Property",
                "value": 10
            },
            "consumption_x": {
                "type": "Property",
                "value": 200
            },
            "id": "urn:ngsi-ld:Light:light2",
            "type": "Light"
        }
    ]
'
```

The broker does not accept this as valid and returns an error:

```json
{
    "@context": "https://uri.etsi.org/ngsi-ld/v1/ngsi-ld-core-context.jsonld",
    "success": [],
    "errors": [
        {
            "entityId": "urn:ngsi-ld:Light:light2",
            "error": {
                "type": "https://uri.etsi.org/ngsi-ld/errors/BadRequestData",
                "title": "invalid entity",
                "detail": "attribute must be a JSON object",
                "status": 400
            }
        }
    ]
}
```

It should be the case that:

-  `@context` **must** be present in each entity if the `Content-Type=application/ld+json`
-  `@context` **must not** be present in any entity if the `Content-Type=application/json`

**Comment by kzangeli** [comment]
Supposedly fixed in PR #389

**Comment by jason-fox** [comment]
Fixed and verified.
